todo_app/app.py

```diff
@@ -1,15 +1,47 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
+from operator import itemgetter
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, get_item, add_item, save_item, remove_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    if request.values.get('sort') == '1':
+        items = sorted(items, key=itemgetter('status'))
+    elif request.values.get('sort') == '2':
+        items = sorted(items, key=itemgetter('status'), reverse=True)
+    else:
+        items = sorted(items, key=itemgetter('id'))
+    return render_template("index.html",items=items)
 
+@app.route('/', methods=['POST'])
+def add_list():
+    newitemtitle = request.form.get('title')
```

Comment:
The Python naming convention is snake_case so this should be `new_item_title`

Reply:
updated to follow python naming convention